### PR TITLE
Include Image Cards for Hobbies + Interest Section, Journey Timeline, and Status Bar

### PR DESCRIPTION
**Pull request by areetaw**
To add more styling, there's now an icon for the tab and a status bar for progress of the user's scrolling. For the `journey` section, a timeline will show up to show different technical experiences. To finish up the web page, the `hobbies + interest` section has image cards.

image

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
...
+
+
+window.onscroll = function() {myFunction()};
+
+function myFunction() {
+  var winScroll = document.body.scrollTop || document.documentElement.scrollTop;
+  var height = document.documentElement.scrollHeight - document.documentElement.clientHeight;
+  var scrolled = (winScroll / height) * 100;
+  document.getElementById("myBar").style.width = scrolled + "%";
+}
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/script.js:38]: It's recommended to end files with newlines

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -78,4 +82,142 @@ a #link {
...
+.progress-container {
+  width: 100%;
+  height: 8px;
+  background: #cccccc;
+}
+
+.progress-bar {
+  height: 8px;
+  background: #384580;
+  width: 0%;
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/style.css:222]: Same here

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+
+window.onscroll = function() {myFunction()};
+
+function myFunction() {
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/script.js:33]: You can rename myFunction to something a bit clearer e.g. onScroll, handleOnScroll, etc.

Comment (areetaw) [reply portfolio/src/main/webapp/script.js]: Fixed, thank you!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (areetaw) [reply portfolio/src/main/webapp/style.css:222]: Gotcha!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (areetaw) [reply portfolio/src/main/webapp/script.js]: Included! Thank you!

**Pull request merged**